### PR TITLE
Install Basics Layout

### PR DESCRIPTION
**Pull request by heintayzar-hm**
I added the following:
Install dependcies
A layout design for projects;
Add normal test cases

### CODE REVIEW

File: src/component/AddBook.jsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  render() {
+    // eslint-disable-next-line no-unused-vars
+    const { addBook } = this.props;
+    return (
+      <>
+        <article data-testid="form-container">
+          <h2>Add New Book</h2>
+          <form>
+            <input data-testid="book-title-input" type="text" placeholder="Book Title" value="" required />
+            <input data-testid="author-title-input" type="text" placeholder="Author" value="" required />
```
Comment (whiz25) [inline src/component/AddBook.jsx:20]: - Please add a submit button. It should not work at this time.

**Review by whiz25 — CHANGES_REQUESTED** [review]
Hi @heintayzar-hm, 

Good job so far! 
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights 🎉 🎉 🎉 
- Codebase is organized 
- Most requirements have been met

## Required Changes ♻️
_Check the comments under the review._
- Add the Category component. It should only have the text "check status"

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

### Martin Afani
#### Slack: whiz25

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/component/AddBook.jsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+class AddBook extends React.Component {
+  constructor(props) {
+    super();
+    this.props = props;
+  }
+
+  render() {
+    // eslint-disable-next-line no-unused-vars
+    const { addBook } = this.props;
+    return (
```
Comment (KayLemba) [inline src/component/AddBook.jsx:14]: [OPTIONAL]It is against microverse policy to disable linters for linters help us write clean code. kindly enable eslint in this file.

File: src/component/Book.jsx [hunk 3]
```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+// eslint-disable-next-line react/prefer-stateless-function
+class Book extends React.Component {
```
Comment (KayLemba) [inline src/component/Book.jsx:5]: [OPTIONAL] Kindly enable eslint

File: src/component/Books.jsx [hunk 4]
```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import AddBook from './AddBook';
+import Book from './Book';
+
+// eslint-disable-next-line react/prefer-stateless-function
+class Books extends React.Component {
+  constructor(props) {
```
Comment (KayLemba) [inline src/component/Books.jsx:7]: Kindly please enable eslint

File: src/component/Category.jsx [hunk 5]
```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+// eslint-disable-next-line react/prefer-stateless-function
+class Category extends React.Component {
+  constructor(props) {
```
Comment (KayLemba) [inline src/component/Category.jsx:5]: [OPTIONAL] Kindly Enable eslint

File: src/component/Header.jsx [hunk 6]
```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+
+// eslint-disable-next-line react/prefer-stateless-function
+class Header extends React.Component {
```
Comment (KayLemba) [inline src/component/Header.jsx:5]: Kindly enable eslint

**Review by KayLemba — APPROVED** [review]
[# STATUS: APPROVED :green_circle: :clap: 

Hello  @heintayzar-hm        , 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

HIGHLIGHTS:

- [x]  Your repo is professional 
- [x] Gitflow is followed
- [x] Descriptive P.R
- [x] Descriptive README file




 

- [x] No Linter Errors







Now you can merge this branch and move on :1st_place_medal: 
Happy Coding cheers 🥇 💯 

well-1

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._]

File: src/component/Book.jsx (hunk 3 above)
Comment (heintayzar-hm) [reply src/component/Book.jsx:5]: I want to use class components. So should I update the linters folder and make a pull request for that?

**Pull request merged**